### PR TITLE
ODC-7280:  Add PipelineRun tab to Dev perspective Pipeline page

### PR DESCRIPTION
**Pull request by vikram-raj**
Story: https://issues.redhat.com/browse/ODC-7280

Description:
- Add PipelineRun tab to Dev perspective Pipeline page

GIF:
Peek 2023-03-23 23-20

### CODE REVIEW

**Comment by vikram-raj** [comment]
cc: @serenamarie125

**Comment by lokanandaprabhu** [comment]
Tested locally. Working fine.

/lgtm


https://user-images.githubusercontent.com/102503482/228740566-920828d4-225d-4731-b220-0446b8e86b80.mov

**Comment by vikram-raj** [comment]
Propagating docs and px acks from the Epic
/label docs-approved
/label px-approved

/retest

**Comment by jerolimov** [comment]
I'm not 100% happy with the e2e test update, but @sanketpathak should decide on that. The rest of the implementation is fine for me. Ping me if you will update the tests and I can re-add lgtm later.

/lgtm

File: frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipelines-runs.ts [hunk 1]
```diff
@@ -254,11 +254,17 @@ Then('user will be redirected to Pipeline Run Details page', () => {
...
+    'PipelineRuns',
+  );
 });
 
 Then('user will remain on pipeline runs page', () => {
-  pipelineRunsPage.verifyTitle();
+  cy.get('.co-m-horizontal-nav__menu-item.co-m-horizontal-nav-item--active > a').should(
+    'have.text',
+    'PipelineRuns',
+  );
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipelines-runs.ts:267]: @vikram-raj I agree we should change `pipelineRunsPage.verifyTitle();` function and use it here as it is

[the remark this answers is not in the document]
Comment (vikram-raj) [reply frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipelines-runs.ts]: Updated it.

**Comment by sanketpathak** [comment]
Verified on cluster bot with admin and non-admin user
Screenshot from 2023-04-01 18-04-39

/label qe-approved

**Pull request merged**